config/samples/swift_v1beta1_swift.yaml

```diff
@@ -18,8 +18,5 @@ spec:
     replicas: 1
     passwordSelectors:
       service: SwiftPassword
-    containerImageAccount: quay.io/podified-antelope-centos9/openstack-swift-account:current-podified
-    containerImageContainer: quay.io/podified-antelope-centos9/openstack-swift-container:current-podified
-    containerImageObject: quay.io/podified-antelope-centos9/openstack-swift-object:current-podified
     containerImageProxy: quay.io/podified-antelope-centos9/openstack-swift-proxy-server:current-podified
```

Comment:
I like that you removed the image parameters above. Why not also get rid of the containerImageProxy and containerImageMemcached here?

Reply:
I was going off of what @kajinamit had said in the first PR attempt that I closed.  I can do as you suggest as long as you tell me its the correct thing to do :-).  Thank you.

Reply:
I will add the dynamic validation in a future PR.